tests/msim/test_msim_EXIT.cpp

```diff
@@ -21,62 +21,99 @@
+                       const Opm::data::Solution& /* sol */,
+                       const std::size_t /* report_step */,
+                       const double /* seconds_elapsed */)
+    {
+        const auto& units = es.getUnits();
+        double water_rate = 0.0;
+        return -units.to_si(Opm::UnitSystem::measure::rate, water_rate);
+    }
 
-double prod_wpr_P1(const EclipseState&  es, const Schedule& /* sched */, const SummaryState&, const data::Solution& /* sol */, size_t /* report_step */, double /* seconds_elapsed */) {
-    const auto& units = es.getUnits();
-    double water_rate = 0.0;
-    return -units.to_si(UnitSystem::measure::rate, water_rate);
-}
+    double prod_wpr_P2(const Opm::EclipseState& es,
+                       const Opm::Schedule& /* sched */,
+                       const Opm::SummaryState&,
+                       const Opm::data::Solution& /* sol */,
+                       const std::size_t report_step,
+                       const double /* seconds_elapsed */)
+    {
+        const auto& units = es.getUnits();
+        double water_rate = 0.0;
+        if (report_step > 5)
+            water_rate = 2.0;  // => WWCT = WWPR / (WOPR + WWPR) = 2/3
 
-double prod_wpr_P2(const EclipseState&  es, const Schedule& /* sched */, const SummaryState&, const data::Solution& /* sol */, size_t report_step, double /* seconds_elapsed */) {
-    const auto& units = es.getUnits();
-    double water_rate = 0.0;
-    if (report_step > 5)
-        water_rate = 2.0;  // => WWCT = WWPR / (WOPR + WWPR) = 2/3
+        return -units.to_si(Opm::UnitSystem::measure::rate, water_rate);
+    }
 
-    return -units.to_si(UnitSystem::measure::rate, water_rate);
-}
+    double prod_wpr_P3(const Opm::EclipseState& es,
+                       const Opm::Schedule& /* sched */,
+                       const Opm::SummaryState&,
+                       const Opm::data::Solution& /* sol */,
+                       const std::size_t /* report_step */,
+                       const double /* seconds_elapsed */)
+    {
+        const auto& units = es.getUnits();
+        double water_rate = 0.0;
+        return -units.to_si(Opm::UnitSystem::measure::rate, water_rate);
+    }
 
-double prod_wpr_P3(const EclipseState&  es, const Schedule& /* sched */, const SummaryState&, const data::Solution& /* sol */, size_t /* report_step */, double /* seconds_elapsed */) {
-    const auto& units = es.getUnits();
-    double water_rate = 0.0;
-    return -units.to_si(UnitSystem::measure::rate, water_rate);
-}
+    double prod_wpr_P4(const Opm::EclipseState& es,
+                       const Opm::Schedule& /* sched */,
+                       const Opm::SummaryState&,
+                       const Opm::data::Solution& /* sol */,
+                       const std::size_t report_step,
+                       const double /* seconds_elapsed */)
+    {
+        const auto& units = es.getUnits();
+        double water_rate = 0.0;
+        if (report_step > 10)
```

Comment:
brace

Reply:
Right.  I've fixed this in the latest update.